tests/common/schedulers/test_sge_commands.py

```diff
@@ -20,9 +22,13 @@
     get_jobs_info,
     get_pending_jobs_info,
 )
-from sqswatcher.sqswatcher import Host
 from tests.common import read_text
 
+# Changed Host class definition in sqswatcher to add gpu info for host.
+# Old Host class definition is included in new definition.
+# Using the old Host class definition in this test should not affect test validity.
+Host = collections.namedtuple("Host", ["instance_id", "hostname", "slots"])
```

Comment:
I'd update the Host initialization below and keep using the Host from sqswatcher rather than a mocked version.